libipl/p10/ipl0.C

```diff
@@ -424,7 +424,7 @@ static int initialize_and_check_clock_chip(uint8_t& clock_count)
 			ipl_log(IPL_ERROR, "pdbg_target_probe is failed for clock '%s', with status = %d\n",
 				pdbg_target_path(clock_target), status);
 
-			ipl_error_callback(IPL_ERR_PDBG_TARGET_NOT_OPERATIONAL);
+			ipl_plat_procedure_error_handler(IPL_ERR_PDBG_TARGET_NOT_OPERATIONAL);
```

Comment:
Is it really a soft error to treat procedure callout as `CODE`? probe means HW is not ready to access @ojayanth I am correct?

Reply:
probe means HW is not ready . I already added comment iniiall related to this . based on the use case need to decide required error or not

Reply:
@rajez133 I don't have a strong opinion on this, you can take a call based on this use case.

Reply:
I have changed this to planar callout instead of procedure callout for this error.